build.rs

```diff
@@ -4,6 +4,11 @@ fn main() {
     let target = env::var("TARGET").unwrap();
 
     if target.starts_with("thumbv6m-") {
-        println!("cargo:rustc-cfg=armv6m")
+        println!("cargo:rustc-cfg=armv6m");
+    } else if target.starts_with("thumbv7m-") {
+        println!("cargo:rustc-cfg=armv7m");
+    } else if target.starts_with("thumbv7em-") {
+        println!("cargo:rustc-cfg=armv7m");
+        println!("cargo:rustc-cfg=armv7em");
```

Comment:
I'd prefer not to enable this cfg if we are not using it right now. It can be left commented out until the need for it arises.